src/hotspot/os_cpu/bsd_zero/orderAccess_bsd_zero.hpp

```diff
@@ -28,8 +28,6 @@
 
 // Included in orderAccess.hpp header file.
 
-#ifdef ARM
```

Comment:
I assume this should have been changed to #if defined(ARM) ??

Reply:
Dang! Right, copy-paste error. Fixed.